pkg/c25519/Makefile

```diff
@@ -0,0 +1,41 @@
+PKG_NAME    = c25519
+PKG_URL     = https://www.dlbeer.co.nz/downloads
+PKG_VERSION = 2017-10-05
+PKG_EXT     = zip
+PKG_LICENSE = PD
+PKG_SHA512  = dbfb4285837ab2ea3d99c448b22877cc7a139ccbaebb1de367e2bec1fd562fe629b389d86603915448078b8fd7e631c8fc9a7d126eb889a1ba0c17611369b190
+
+PKG_SHAFILE ?= $(PKGDIRBASE)/$(PKG_NAME)-sha512.txt
+PKG_BUILDDIR ?= $(PKGDIRBASE)/$(PKG_NAME)
+
+ifneq ($(RIOTBASE),)
+include $(RIOTBASE)/Makefile.base
+endif
+
+.PHONY: all clean distclean
+
+prepare: $(PKG_BUILDDIR)/
+
+all: $(PKG_BUILDDIR)/
+	"$(MAKE)" -C $(PKG_BUILDDIR)/src -f $(CURDIR)/Makefile.c25519
+
+$(PKGDIRBASE):
+	mkdir -p $(PKGDIRBASE)
+
+$(PKG_SHAFILE): $(PKGDIRBASE)
+	$(Q)echo "$(PKG_SHA512) $(PKGDIRBASE)/$(PKG_NAME)-$(PKG_VERSION).$(PKG_EXT)" > $@
```

Comment:
Was there a cleaner way of creating this file than using `echo`?

Reply:
```test "$(PKG_SHA512) $(FILENAME)" = "$(sha512sum $FILENAME)"```

Don't forget double dollars to make make happy.
Maybe do ```export F=$(SUUUUUUUUUUUUUUUUUUUUUUUUUUPERLONGFILENAMEVARIABLE); test "$(SHA512) $$(F)" = ... ```

Reply:
Something like 9430b2b2d2?